src/index.html

```diff
@@ -5,8 +5,270 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+        src="/images/logo.png"
+        alt="moyo_logo"
+        class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_link is-active">APPLE</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">SAMSUNG</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">ФОТО</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+
+
+
+    <main class="main">
+
+      <div class="card" data-qa="card">
+        <img
+          src="/images/imac.jpeg"
+          class="card__photo"
+          alt="iMac"
+        >
+        </img>
+        <h2 class="card_text--name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+        <p class="card_text--code">Код товара: 195434</p>
+        <div class="card__stars stars--4">
+          <div class="card__stars__item "></div>
```

Comment:
this is not valid BEM
block__element--modifier
or 
block__element-name--modifier